docs/source/contributing.rst

```diff
@@ -0,0 +1,283 @@
+Contributing to xyzspaces
+=========================
+
+Overview
+--------
+
+Contributions to xyzspaces are very welcome.  They are likely to
+be accepted more quickly if they follow these guidelines.
+
+Below are guidelines, when submitting a pull request:
+
+- All existing tests should pass. Please make sure that the test
+  suite passes, both locally and on
+  `Travis CI <https://travis-ci.com/github/heremaps/xyz-spaces-python>`_.  Status on
+  Travis will be visible on a pull request.
+
+- New functionality should include tests.  Please write reasonable
+  tests for your code and make sure that they pass on your pull request.
+
+- Classes, methods, functions, etc. should have docstrings.  The first
+  line of a docstring should be a standalone summary.  Parameters and
+  return values should be documented explicitly.
+
+- Follow PEP 8 when possible. We use `Black
+  <https://black.readthedocs.io/en/stable/>`_,  `Flake8
+  <http://flake8.pycqa.org/en/latest/>`_, `isort <https://pypi.org/project/isort/>`_, `typing <https://pypi.org/project/typing/>`_ and `darglint <https://pypi.org/project/darglint/>`_ to ensure a consistent code
+  format throughout the project. For more details see
+  :ref:`below <contributing_style>`.
+
+- Imports should be grouped with standard library imports first,
+  3rd-party libraries next, and xyzspaces imports third.  Within each
+  grouping, imports should be alphabetized.  Always use absolute
+  imports when possible, and explicit relative imports for local
+  imports when necessary in tests.
+
+- xyzspaces supports Python 3.6+.
+
+
+Seven Steps for Contributing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are seven basic steps to contributing to *xyzspaces*:
+
+1) Fork the *xyzspaces* git repository
+2) Create a development environment
+3) Install *xyzspaces* dependencies
+4) Make changes to code and add tests
+5) Run liniting
```

Comment:
Typo?

Reply:
https://code.visualstudio.com/docs/python/linting#:~:text=Linting%20highlights%20syntactical%20and%20stylistic,that%20can%20lead%20to%20errors.

Reply:
fixed typo.